### DISCUSSION TITLE
Omega output on model levels

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1327] [Q&A] [answered]
Is it possible to output omega (pa/sec) on model levels in addition to w (m/sec)? Or is omega only available via post-processing? It used to be possible to output omega directly on a selection of pressure levels in the atmos 4x daily files, but my understanding is that mode of output files has been deprecated.

**Comment by junwang-noaa** [comment]
@benjamin-cash The omega field can be computed as a post product in the model and output directly on the native grid from the model.  I think you can still output omega through atmos 4x daily files. We are outputting the model fields in other output grids (e.g. Gaussian grid) in history files and call post processing outside the forecast to compute omega. @WenMeng-NOAA can we output omega on model levels?

**Comment by WenMeng-NOAA** [comment, marked as answer]
Omega is computed in UPP. It can be output from UPP either on model levels and pressure levels.

**Reply by benjamin-cash** [reply]
Thanks! Just to confirm, it is possible to output omega on pressure levels on the model grid in the 4x_daily files (which I was told in #1056 is being deprecated), but not in the gaussian netcdf history files?

**Reply by junwang-noaa** [reply]
That part of code should still work, but it is not used/tested in global workflow. As Wen mentioned, you can also output the omega field through inline post.

**Reply by benjamin-cash** [reply]
@junwang-noaa @WenMeng-NOAA - Do you happen to know if the algorithm UPP uses to interpolate from model levels to pressure is documented anywhere? I've never been entirely clear on what the model levels in the dynf (atmf) files represent, given the nature of FV3, so I'm not entirely sure how that interpolation is done.
